### ISSUE TITLE
Enforced style EmptyLinesAroundBlockBody seeming to not work as expected?

### ISSUE THREAD

**Issue by FeminismIsAwesome** [bbatsov/rubocop#3714]
*Be clear, concise and precise in your description of the problem.
Open an issue with a descriptive title and a summary in grammatically correct,
complete sentences.*

*Use the template below when reporting bugs. Please, make sure that
you're running the latest stable RuboCop and that the problem you're reporting
hasn't been reported (and potentially fixed) already.*

*Before filing the ticket you should replace all text above the horizontal
rule with your own words.*

--------

## Expected behavior

in my rubocop.yml, I have the following rule configured:

```
Style/EmptyLinesAroundBlockBody:
  EnforcedStyle: no_empty_lines
```

And have code like such:
```
def test
  puts "You are in the method"
  yield
  puts "You are again back to the method"
  yield
end

test do  puts "you are in the block" end
test do
  puts "you are in the block"
end

```

I expect it to be marking a warning/notification that a rule has been violated in this situation. This particular code was written after noticing if I have two it blocks together in a spec, it doesn't have an issue with that rule turned on:

like:
```
it "should test something" do
  # validate something
end
it "should test something else" do
 #validate something
end
```
Which I thought would flag in a similar fashion
## Actual behavior

It says no offenses are detected. Is this not actually what the rule does? Is there a different rule I should be using for that kind of behavior?

 rubocop version: 0.45.0
 (running on ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin14]
)

**Comment by FeminismIsAwesome** [comment]
I did some more research and thought about this and so decided to close.
